src/civic/README.md

```diff
@@ -14,3 +14,14 @@ The `variant.js` module handles all parsing/processing of variant records in CIV
 
 In general this loader attempts to update existing records based on their CIViC evidence ID, however when the mapping from CIViC evidence record to GraphKB statement is not 1 to 1 then the
 statement is sometimes recreated and the old version soft-deleted instead since we cannot resolve tracking between the 2 for these cases.
+
+### Trusted Curators
+
+By default this loader only loads accepted (reviewed and approved) evidence items from CIViC. However,
```

Comment:
the max line width in this section seems to be different from the width in the 'About' section

Reply:
I had a "soft" max line width of 100 characters but i found that I would have to just re-do the entire paragraph when a word was added so I have stopped using them in md

Reply:
I've removed all the max-width breaks